metadata-ingestion/src/datahub/utilities/perf_timer.py

```diff
@@ -57,29 +57,34 @@ def __exit__(
             self.finish()
         return None
 
-    def elapsed_seconds(self) -> float:
+    def elapsed_seconds(self, digits: Optional[int] = None) -> float:
```

Comment:
let's default to 2 digits here

```suggestion
    def elapsed_seconds(self, digits: int = 2) -> float:
```